shopinvader_validation_token/services/customer_service.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _, exceptions
+from odoo.addons.component.core import Component
+
+
+class ShopinvaderCustomerService(Component):
```

Comment:
I do not understand this. Why do you impact the create depending on a guest feature?
If you want to allow that it should be possible without activating the guest mode which is not like you did

Reply:
This feature is not only for guest.
Depending on the configuration/notification, it's available for `guest`, `customer` or both.

As a `guest` service inherits `customer` service, it's applied on both services.